examples/bid-optimizer/src/MyPluginImpl.ts

```diff
@@ -1,13 +1,13 @@
-import { core } from "@mediarithmics/plugins-nodejs-sdk";
+import {core} from '@mediarithmics/plugins-nodejs-sdk';
 
 export class MyBidOptimizerPlugin extends core.BidOptimizerPlugin {
   protected onBidDecisions(
     request: core.BidOptimizerRequest,
     instanceContext: core.BidOptimizerBaseInstanceContext
   ): Promise<core.BidOptimizerPluginResponse> {
-    
+
     // Optimization, we only do the stringify  if we are really on debug / silly mode
-    if (this.logger.level === "debug" || this.logger.level === "silly") {
+    if (this.logger.level === 'debug' || this.logger.level === 'silly') {
```

Comment:
Using the "best in class" logger would be better than leaving those hacks (we know how to write better code now).

```
this.logger.debug(
        `Received inside plugin: %j`, request
      );
```

Reply:
You're right, but this is not the goal of this PR. I haven't reviewed the whole code. Just changed why I wanted to improve.
It could be done in another one.